core/domain/certificate_assessment_services_test.py

```
@@ -194,6 +236,203 @@ def test_get_update_and_delete_certificate_assessment_offering(
+    ) -> None:
+        offering = certificate_assessment_services.create_certificate_assessment_offering(
+            title='History',
+            description='desc',
+            classroom_id=self.classroom_id,
+            topic_ids=[self.topic_id],
+            total_questions=5,
+            time_limit_in_minutes=30,
+            demonstrates=['Skill'],
+            async_status='Available',
+        )
+        started_at = datetime.datetime(2026, 1, 2, 3, 4, 5)
```

Ruff (0.16.0)

[warning] `datetime.datetime()` called without a `tzinfo` argument

(DTZ001)